lib/src/HttpUtils.cc

```diff
@@ -603,4 +603,28 @@ ContentType parseContentType(const string_view &contentType)
     return iter->second;
 }
 
+FileType parseFileType(const std::string &fileExtension)
+{
+    // https://en.wikipedia.org/wiki/List_of_file_formats
+    static const std::unordered_map<string_view, FileType> map_{
+        {"html", FT_DOCUMENT},
+        {"docx", FT_DOCUMENT},
+        {"zip", FT_ARCHIVE},
+        {"rar", FT_ARCHIVE},
+        {"xz", FT_ARCHIVE},
+        {"mp3", FT_AUDIO},
+        {"wav", FT_AUDIO},
+        {"avi", FT_MEDIA},
+        {"gif", FT_MEDIA},
+        {"mp4", FT_MEDIA},
+        {"mov", FT_MEDIA},
+        {"png", FT_IMAGE},
+        {"jpeg", FT_IMAGE},
+        {"ico", FT_IMAGE}};
```

Comment:
Lazy to add the rest, I only put what is most commonly used nowadays, maybe soon I will add more.

Reply:
And maybe change it to a map? For example, the key would be the file type and the values would be a string vector with all possible types?

Reply:
The current map is good for getting a type from an extension string.